setup.cfg

```diff
@@ -51,14 +51,13 @@ python_requires = >= 3.7
 packages =
 
 [flake8]
+# https://flake8.readthedocs.org/en/latest/
+exclude=\
+    venv*/*,\
+    local/*,\
+    build/*,\
+    .eggs/*,\
+    .tox/*
```

Comment:
Similarly, flake8 should always be run through pre-commit, so these are unnecessary.